instance/models/instance.py

```diff
@@ -204,7 +204,7 @@ class GitHubInstanceMixin(VersionControlInstanceMixin):
     """
     github_organization_name = models.CharField(max_length=200, db_index=True)
     github_repository_name = models.CharField(max_length=200, db_index=True)
-    github_pr_number = models.IntegerField(blank=True, null=True)
+    github_pr_url = models.CharField(max_length=200, blank=True, default='')
```

Comment:
@antoviaque It appears you're correct. I'm thinking of behavior I ran into when working with Django 1.4.